cli/cmd/uninstall.go

```diff
@@ -196,31 +196,61 @@ func getWorkloadRolloutJsonPatch(obj client.Object, pts *v1.PodTemplateSpec) ([]
 		}
 	}
 
+	// read the original env vars (of the manifest) from the annotation
+	var origManifestEnv map[string]map[string]string
+	if obj.GetAnnotations() != nil {
+		manifestEnvAnnotation := obj.GetAnnotations()[consts.ManifestEnvOriginalValAnnotation]
+		err := json.Unmarshal([]byte(manifestEnvAnnotation), &origManifestEnv)
```

Comment:
will it err if the `manifestEnvAnnotation` is an empty string? which is the case if we did not change any env